spec/models/export/template_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Export::Template do
+  describe '.source_field_for' do
+    subject { Export::Template.source_field_for(dest_field_name, framework_short_name) }
```

Comment:
it doesn't look like this subject is being used. is it redundant?

Reply:
It's being used on line 20 with the `it { is_expected.to <> }`

Reply:
perhaps this spec would be easier to follow if the call was explicit in the one expectation it is used?

Reply:
I do like `subject` as a cue for the reader – it's says that this is the point of the thing – `it` – for all tests here, and then when `context` needs to vary a param it can, keeping the thing that changed obvious in the let and how it affected the subject right next to it in the commit. I agree that it's probably been used pre-emptively and habitually here for two cases and is also covered in the `describe`, so it's probably my preference getting in the way of understanding.